pages/linux/pwgen.md

```diff
@@ -0,0 +1,7 @@
+# pwgen
+
+> Generate passwords.
+
+- Generate 8 characters long random passwords with symbols :
+
+`pwgen -sy`
```

Comment:
As per: http://linux.die.net/man/1/pwgen, you're missing the length (8). So a few samples could be:

```md
- Generate random password with symbols:

`pwgen -y {{length}}`

- Generate hard-to-memorize passwords:

`pwgen -s {{length}}`
```